javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,93 @@
+            let text1="Your dog will be "+adogYear+" dog years old in "+dogYearFuture;
+            document.getElementById("dogAgeCount").innerHTML=text1;
+
+           }else{
+            let text1="Your dog will be "+adogYear+" human years old in "+dogYearFuture;
+            document.getElementById("dogAgeCount").innerHTML=text1;
+
+           }
+        }
+/* *********************************************House Price Estimator************************************************************ */
+
+                                 /* ************************Peter****************************/
+
+        let widthPeter =8;
+        let deepPeter = 10;
+        let heightPeter =10;
+        let gardenSizeInM2=100;
+        let hunderedM2Price =2500000;
+        let volumeInMeters =widthPeter*deepPeter*heightPeter;
+        let housePricePeter = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+        //console.log(housePricePeter);
+        if( hunderedM2Price>=housePricePeter){
+            let text3 ="Actural price for 100m2 house is "+hunderedM2Price+" <br>Peter is paying "+housePricePeter+ " which much less .";
+            document.getElementById("peterPrice").innerHTML=text3;
+        }else{
+            let  text3 ="Actural price for 100m2 house is "+hunderedM2Price+" <br>Peter is paying "+housePricePeter+ " which much more .";
+            document.getElementById("peterPrice").innerHTML=text3;
+        }
+
+
+
+                       /* ************************Julia****************************/
+
+        const widthJulia =5;
+        const deepJulia = 11;
+        const heighJulia =8;
+        const gardnSizeInM2=70;
+        const seventyM2Price =1000000;
+        const volumeInMetersJulia =widthJulia*deepJulia*heighJulia;
+        const housePriceJulia = volumeInMetersJulia * 2.5 * 1000 + gardnSizeInM2 * 300;
+       // console.log(housePriceJulia);
+        if( seventyM2Price>=housePriceJulia){
+            let text4 ="Actural price for 70m2 house is "+seventyM2Price+" <br>Julia is paying "+housePriceJulia+ " which much less .";
+            document.getElementById("juliaPrice").innerHTML=text4;
+        }else{
+            let  text4 ="Actural price for 70m2 house is "+seventyM2Price+" <br>Julia is paying "+housePriceJulia+ " which much more .";
+            document.getElementById("juliaPrice").innerHTML=text4;
+        }
+
+        /* *********************************************Ez Namey (Startup name generator)************************************************************ */
+        
+        function ezNamy(){
+            let firstWords= ["Mindstuff","Black","Tech","Diamondback","Jumping","Great","Industry","Zest","Teletrans","Melt"];
+            let secondWords=["Corporation","Soapdots App","Slantwise Ltd","Care","Systems","Strut",".Al","Woodpecker Care","Inc","Technologies"];
+            let random1= Math.floor(Math.random() * 10) + 0;
+            let startupName =firstWords[random1] +" "+ secondWords[random1];
```

Comment:
Use one random index will get you the results needed, but how about using two? Won't that increase the pool of random startup names to pick from? What do you think?

Reply:
Hi Ramin actually i want same array index for both arrays.. because the startup name have two parts..firstname in first array and second part in second array... and they are in same sequence.. so i am taking same index for both array to get correct name of startup

Reply:
Create two arrays called firstWords, secondWords. The arrays should have 10 elements containing strings of possible startup names.

Reply:
Sure, no worries. Just making sure you know this extra piece of info. It depends on the contents of your array. For example, the below arrays can yield `"Happy Help"` and `"Happy Company"`, and both would be valid names.

```js
let firstWords = ["Happy"];
let secondWords = ["Help", "Company"];
```